sdks/java/core/src/test/java/org/apache/beam/sdk/io/CompressedSourceTest.java

```diff
@@ -161,6 +189,28 @@ public void testGzipSplittable() throws Exception {
     assertFalse(source.isSplittable());
   }
 
+  /** Test splittability of files in LZO mode -- none should be splittable. */
+  @Test
+  public void testLzoSplittable() throws Exception {
```

Comment:
Please add a similar test for `testLzopSplittable()`.

Reply:
Thanks for pointing this out, this has been added.